fastly/fixtures/erls/cleanup.yaml

```diff
@@ -0,0 +1,45 @@
+---
+version: 1
+interactions:
+- request:
+    body: ""
+    form: {}
+    headers:
+      User-Agent:
+      - FastlyGo/6.2.0 (+github.com/fastly/go-fastly; go1.17.6)
+    url: https://api.fastly.com/rate-limiters/wBT3mVJW6QVW0omIYSiyF
+    method: DELETE
+  response:
+    body: '{"msg":"Record not found","detail":"Cannot find vclmodule ''''"}'
+    headers:
+      Accept-Ranges:
+      - bytes
+      Cache-Control:
+      - no-store
+      Content-Type:
+      - application/json
+      Date:
+      - Thu, 10 Mar 2022 01:29:45 GMT
```

Comment:
Can we update the fixtures using the latest version of the code (as a fair amount of refactoring has occurred).